File: src/main/resources/io/bgplayground/cooljava/scripts/coffee.py
python_phase = False

def move_coffee(event):
    global python_phase
    python_phase = not python_phase
    document.getElementById("coffee").setAttribute(
        "class", "bounce-a" if python_phase else "bounce-b")
    document.getElementById("status").setTextContent("GraalPy → W3C DOM")

document.getElementById("run-python").addEventListener("click", move_coffee, False)

File: src/main/resources/io/bgplayground/cooljava/scripts/coffee.js
let jsPhase = false;
document.getElementById('run-js').addEventListener('click', function(event) {
    jsPhase = !jsPhase;
    document.getElementById('coffee').setAttribute('class', jsPhase ? 'wave-a' : 'wave-b');
    document.getElementById('status').setTextContent('GraalJS → W3C DOM');
}, false);
